src/components/appfooter.main.jsx

```diff
@@ -23,73 +23,52 @@ import React from 'react';
 import intl from 'react-intl-universal';
 import { Link } from 'react-router-dom';
 
+import './appfooter.main.less';
```

Comment:
i think we should keep this import above with the rest. is there a particular reason why you've left a blank line in-between?

Reply:
It's a different type of import, so it made sense to separate it from the rest with an empty line. I can remove it, it's arbitrary.